mautic/mautic
# Fix email_queued search query

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | y
| New feature/enhancement? (use the a.x branch)      | n
| Deprecations?                          | n
| BC breaks? (use the c.x branch)        | n
| Automated tests included?              | modified existing
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | #10748



#### Description:
The email_queued filter does not show queued emails. At this moment the query is expecting queued emails with status `sent`, but should `pending` or `rescheduled`.



#### Steps to test this PR:
1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Set limit for messages, e.g. 1 message per day.
3. Send 2 marketing emails to same contact
4. On the emails list the second email should be queued, click on the `1 Queued` stat to view the contact list
5. Queued contact should appear on the list




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10803"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
